### PR TITLE
🌱 added variable example and description to clusterclass yamls

### PR DESCRIPTION
**Pull request by killianmuldoon**
Signed-off-by: killianmuldoon <kmuldoon@vmware.com>

Add variable examples and description and patch descriptions to the files used in the ClusterClass quickstart end to end tests and the clusterclass-quick-start.yaml docker template.

### CODE REVIEW

**Comment by fabriziopandini** [comment]
/lgtm
@sbueringer PTAL

File: test/e2e/data/infrastructure-docker/v1beta1/clusterclass-quick-start.yaml [hunk 1]
```diff
@@ -39,20 +39,27 @@ spec:
...
         default: "k8s.gcr.io"
+        example: "k8s.gcr.io"
+        description: "imageRepository is the container registry to pull images from. If empty, `k8s.gcr.io` will be used by default."
   - name: etcdImageTag
     required: true
     schema:
       openAPIV3Schema:
         type: string
         default: ""
+        example: "3.5.1"
```
Comment (sbueringer) [inline test/e2e/data/infrastructure-docker/v1beta1/clusterclass-quick-start.yaml:50]: Let's use `3.5.1-0`, `3.5.1` does not exist. (full image for tests: `k8s.gcr.io/etcd:3.5.1-0`)
```suggestion
        example: "3.5.1-0"
```

File: test/e2e/data/infrastructure-docker/v1beta1/clusterclass-quick-start.yaml [hunk 2]
```diff
@@ -39,20 +39,27 @@ spec:
...
+        description: "etcdImageTag is the tag for the etcd image."
   - name: coreDNSImageTag
     required: true
     schema:
       openAPIV3Schema:
         type: string
         default: ""
+        example: "1.8.5"
```
Comment (sbueringer) [inline test/e2e/data/infrastructure-docker/v1beta1/clusterclass-quick-start.yaml:58]: We have to have a `v` prefix. Let's also use v1.8.4 as that's the current default (full image `k8s.gcr.io/coredns/coredns:v1.8.4`)
```suggestion
        example: "v1.8.4"
```

File: test/e2e/data/infrastructure-docker/v1beta1/clusterclass-quick-start.yaml (hunk 1 above)
Comment (sbueringer) [reply test/e2e/data/infrastructure-docker/v1beta1/clusterclass-quick-start.yaml:50]: Same in the other ClusterClass

File: test/e2e/data/infrastructure-docker/v1beta1/clusterclass-quick-start.yaml (hunk 2 above)
Comment (sbueringer) [reply test/e2e/data/infrastructure-docker/v1beta1/clusterclass-quick-start.yaml:58]: Same in the other ClusterClass

**Review by sbueringer — COMMENTED** [review]
Thx for picking that one up!

Already shows that descriptions and examples have a benefit I think!

**Comment by sbueringer** [comment]
Thx!
/lgtm

**Pull request merged**